Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityPowerMockTest.java

```diff
@@ -104,4 +107,18 @@ public void onPaymentMethodSelected_startsAddCardActivity() {
         IntentForResult intent = shadowActivity.peekNextStartedActivityForResult();
         assertEquals(AddCardActivity.class.getName(), intent.intent.getComponent().getClassName());
     }
+
+    @Test
+    public void onConfigurationFetch_whenDisabledAndroidPay_doesNotCallAndroidPay() {
```

Comment:
This test is currently failing on Travis because we do skip `AndroidPay.isReadyToPay` but then it tries to call on a method on a view which hasn't been setup in these tests.

Lets remove this test and use the one I suggested in `DropInActivityUnitTest.java`